tests/playwright/src/model/pages/run-image-page.ts

```diff
@@ -81,7 +94,25 @@ export class RunImagePage extends BasePage {
       throw Error(`Start Button not enabled: ${errMessage}`);
     }
     await this.startContainerButton.click();
-    return new ContainersPage(this.page);
```

Comment:
I don't really agree with removing this, it's completely pattern breaking to have a method that does part of its functionality screen navigation to return void.